Refresh translations for JA & KO

johndmulhausen opened the pull request:
What it says on the tin. JA & KO will capture goodies like the inference service docs, refreshed reference guides, and much more. 

<!-- preview-links-comment -->

📄 **[View preview links for changed content](https://github.com/wandb/docs/pull/1499#issuecomment-3208066763)**

mdlinville committed:
[DOCS-1136] Rename guide about creating experiments

johndmulhausen committed:
Refresh translation stubs

tcapelle committed:
gpt4.1 translation

tcapelle committed:
korean gpt4.1 v4

mdlinville committed:
Merge branch 'main' into DOCS-1136

mdlinville commented on content/ja/_includes/enterprise-cloud-only.md:
```diff
@@ -1,2 +1,5 @@
-This feature requires a [Pro or Enterprise plan](https://wandb.ai/site/pricing/).
+---
+{}
```
I have never seen this syntax, and it isn't present in the English versions. What is it about?

tcapelle replied:
this is probably a parasite of my poor handling of pages without header

johndmulhausen replied:
Removed

mdlinville commented on content/ja/guides/core/artifacts/create-a-new-artifact-version.md:
````diff
@@ -1,140 +1,140 @@
     run.upsert_artifact(artifact, distributed_id="my_dist_artifact")
 ```
 
 #### Run 3
````
```suggestion
#### Run 3:
```
This should be fixed in the English and Korean too.

mdlinville reviewed (changes requested):
Partial review from the top through ja Registry.

mdlinville commented on content/ja/guides/hosting/hosting-options/self-managed/install-on-public-cloud/_index.md:
```diff
@@ -1,10 +1,10 @@
     identifier: ja-guides-hosting-hosting-options-self-managed-install-on-public-cloud-_index
     parent: self-managed
-url: /ja/guides/hosting/self-managed/install-on-public-cloud
+url: guides/hosting/self-managed/install-on-public-cloud
```
Just verifying that this change is corrrect?

mdlinville commented on content/ja/guides/hosting/iam/access-management/manage-organization.md:
```diff
@@ -1,333 +1,323 @@
-1. 役割が持つ許可を要約している **カスタム役割の許可** セクションを確認します。
-1. **役割を作成** をクリックします。
+{{% tab header="マルチテナント SaaS Cloud" value="saas" %}}
+1. https://wandb.ai/home にアクセスします。
```
I wonder why this list and the one a few lines down got re-numbered with explicit numbers.

mdlinville commented on content/ja/guides/hosting/iam/access-management/manage-organization.md:
```diff
@@ -1,333 +1,323 @@
 
-1. https://wandb.ai/home に移動します。
-2. ページの右上隅にある **ユーザーメニュー** ドロップダウンを選択します。ドロップダウンから **ユーザー** を選択します。
-4. 検索バーにユーザーの名前またはメールアドレスを入力します。
```
Good catch

mdlinville reviewed (changes requested):
Partial review through ja/guides/inference.

mdlinville commented on content/ja/guides/integrations/docker.md:
```diff
@@ -1,6 +1,8 @@
 ---
-title: Docker
-description: W&B を Docker と統合する方法。
+title: 'Docker
```
Something wrong here.

mdlinville commented on content/ja/guides/integrations/mmengine.md:
```diff
@@ -1,33 +1,35 @@
 ---
-title: MMEngine
+title: 'MMEngine
```
Something wrong here.

mdlinville commented on content/ja/guides/integrations/torchtune.md:
```diff
@@ -1,5 +1,5 @@
 ---
-title: Pytorch チューニングする torchtune
+title: Pytorch torchtune
```
Nit: I noticed that some of these pages have no description set, probably in the English too.

mdlinville reviewed (changes requested):
Partial review through content/ja/guides/integrations/yolox.md. I'll keep going tomorrow.

mdlinville committed:
Merge branch 'main' into DOCS-1136

mdlinville committed:
Merge remote-tracking branch 'origin/DOCS-1136' into ja-ko-refresh

mdlinville committed:
Merge branch 'main' into ja-ko-refresh

mdlinville commented on content/ja/guides/models/_index.md:
```diff
@@ -1,21 +1,21 @@
 ---
-title: W&B モデル
+title: W&B Models
```
Is this correct?

mdlinville commented on content/ja/guides/models/app/features/panels/_index.md:
```diff
@@ -1,217 +1,228 @@
 ---
 
-ワークスペースパネルの可視化機能を使用して [ログされたデータ]({{< relref path="/ref/python/log.md" lang="ja" >}}) をキーごとに探り、ハイパーパラメーターと出力メトリクスの関係を可視化することなどができます。
+workspace パネルの可視化を使って、[記録したデータ]({{< relref path="/ref/python/sdk/classes/run.md/#method-runlog" lang="ja" >}})をキーごとに探索し、ハイパーパラメーターと出力メトリクスの関係を可視化するなど、さまざまな用途でご利用いただけます。
```
This looks off.

mdlinville commented on content/ja/guides/models/app/features/panels/_index.md:
```diff
@@ -1,217 +1,228 @@
 {{% /alert %}}
 
-## ワークスペースをリセットする方法
+## workspace をリセットする
```
This looks off.

mdlinville commented on content/ja/guides/models/app/settings-page/team-settings.md:
```diff
@@ -1,6 +1,6 @@
-title: チーム設定を管理する
-description: チーム設定ページでチームのメンバー、アバター、アラート、プライバシー設定を管理します。
+title: チーム設定を管理
+description: チームのメンバー、アバター、アラート、およびプライバシー設定は、Team Settings ページで管理できます。
```
This looks like it might be off.

mdlinville reviewed (changes requested):
Reviewed through content/ja/guides/models/app/settings-page/user-settings.md. I also merged a change into this branch before it was merged to `main` so I could see and resolve related conflicts.

olachinkei committed:
update table translation

olachinkei committed:
edit inference Japanese

olachinkei committed:
edit Japanese in core

olachinkei committed:
edit Japanese in inference

olachinkei committed:
edit Japanese in core

johndmulhausen committed:
Update Korean automation guides to use consistent terminology and remove unnecessary content

johndmulhausen committed:
Remove empty front-matter from JA

johndmulhausen committed:
Update content/ja/guides/core/artifacts/create-a-new-artifact-version.md

Co-authored-by: Matt Linville <matt.linville@wandb.com>

johndmulhausen committed:
Update Japanese and Korean guides to standardize terminology and improve clarity in registry and automation sections

mdlinville committed:
Merge branch 'main' into ja-ko-refresh

johndmulhausen committed:
Standardize terminology from "자동화" to "Automation" across Korean documentation for consistency and clarity.

johndmulhausen committed:
Standardize terminology to "Automation" in Korean documentation and update titles for consistency across automation guides.

johndmulhausen committed:
Update Japanese documentation for consistency and clarity

- Add newline at the end of `enterprise-cloud-only.md`
- Remove unnecessary formatting in artifact version examples
- Standardize titles and descriptions in Docker, MMEngine, and PyTorch torchtune integration guides
- Fix links in workspace panel guide for better navigation

johndmulhausen commented:
Closing as we've totally changed docs platforms.
